src/components/Search/TimeRangeDropdown.tsx

```diff
@@ -65,9 +88,17 @@ function TimeRangeDropdown({
             <MenuItem
               value={value}
               key={value}
-              className={`timerange-dropdown-item ${menuItemStyles}`}
+              className={`timerange-dropdown-item ${menuItemStyles} ${styles.menuItem}`}
             >
-              {text}
+              <div className={styles.dateRange}>
```

Comment:
you don't need the extra div, just add the `space-between` to the MenuItem directly.

Reply:
thanks for the explanation, let's keep it like you did it, but please add a comment before line 47 with this explanation:
```
// This extra flexible div is needed, so that the information looks the same in 
// both the MenuItem (in the dropdown options) and the SelectedItem (in
// the dropdown, when the dropdown is closed).
```